Source/WebInspectorUI/UserInterface/Views/SettingsTabContentView.js

```diff
@@ -414,6 +414,11 @@ WI.SettingsTabContentView = class SettingsTabContentView extends WI.TabContentVi
             experimentalSettingsView.addSeparator();
         }
 
+        let sourcesGroup = experimentalSettingsView.addGroup(WI.unlocalizedString("Sources:"));
+        sourcesGroup.addSetting(WI.settings.experimentalAggressivelyLimitSourceCodeHighlighting, WI.unlocalizedString("Aggresively limit source code line length limit for highlighting"));
```

Comment:
NIT: I dont think "Aggressively" really adds much here beyond what "Limit" would already imply.  how about something simpler like "Limit syntax highlighting on long lines of code"?